controllers/component_build_controller_test.go

```diff
@@ -246,6 +263,12 @@ var _ = Describe("Component initial build controller", func() {
 			}
 			Expect(k8sClient.Create(ctx, gitSecret)).Should(Succeed())
 
+			// Configure the build bundle
```

Comment:
I'd create a separate test Context where only build bundle cases (3 of them) are tested in a separate test cases.